pkg/registry/apis/query/parser.go

```diff
@@ -161,15 +161,15 @@ func (p *queryParser) parseRequest(ctx context.Context, input *query.QueryDataRe
 				if !ok {
 					target, ok = expressions[refId]
 					if !ok {
-						return rsp, fmt.Errorf("expression [%s] is missing variable [%s]", exp.RefID, refId)
+						return rsp, makeDependencyError(exp.RefID, refId)
```

Comment:
Nit. These sse validations should probably be moved to the sse package? We can revisit later maybe

Reply:
unclear where the best home is -- the MT query service is implementing query parsing and orchestration directly.  In the `expr` package it is managed by the expression engine itself.

But yes -- we can continue to evolve and revisit error support in general

Reply:
Sounds good 👍